pipeline/plot/unit_characteristic_plot.py

```diff
@@ -308,6 +310,95 @@ def plot_unit_bilateral_photostim_effect(probe_insertion, clustering_method=None
     return fig
 
 
+def plot_pseudocoronal_slice(probe_insertion, shank_no=1):
+    probe_insertion = probe_insertion.proj()
+
+    # ---- Electrode sites ----
+    annotated_electrodes = (lab.ElectrodeConfig.Electrode * lab.ProbeType.Electrode
+                            * ephys.ProbeInsertion
+                            * histology.ElectrodeCCFPosition.ElectrodePosition
+                            & probe_insertion & {'shank': shank_no})
+
+    coords = np.array(list(zip(*annotated_electrodes.fetch('ccf_z', 'ccf_y', 'ccf_x'))))  # (AP, DV, ML)
+
+    # ---- Labeled Probe Track points ----
+    prb_trk_coords = np.array(list(zip(*(histology.LabeledProbeTrack.Point
+                                         & probe_insertion & {'shank': shank_no}).fetch('ccf_z', 'ccf_y', 'ccf_x'))))
+
+    # ---- linear fit of probe in DV-AP axis ----
+    X = np.asmatrix(np.hstack((np.ones((coords.shape[0], 1)), coords[:, 1][:, np.newaxis])))  # DV
+    y = np.asmatrix(coords[:, 0]).T  # AP
+    XtX = X.T * X
+    Xty = X.T * y
+    b = np.linalg.solve(XtX, Xty)
+
+    # ---- predict x coordinates ----
+    voxel_res = _ccf_vox_res or _get_ccf_vox_res()
+    lr_max, dv_max, _ = _ccf_xyz_max or _get_ccf_xyz_max()
+
+    dv_coords = np.arange(0, dv_max, voxel_res)
+
+    X2 = np.asmatrix(np.hstack((np.ones((len(dv_coords), 1)), dv_coords[:, np.newaxis])))
+
+    ap_coords = np.array(X2 * b).flatten().astype(np.int)
+
+    # round to the nearest voxel resolution
+    ap_coords = (voxel_res * np.round(ap_coords / voxel_res)).astype(np.int)
+
+    # -- inspect the fit --
+    if True:
```

Comment:
is the 'if' leftover debug code, etc?

Reply:
ah yeah true, yes yes left over from debug, I'll delete